README.md

```diff
@@ -41,6 +41,61 @@ $ pip3 install mythril
 
 If you plan to analyze Solidity code you'll also need the [native version of solc](http://solidity.readthedocs.io/en/v0.4.21/installing-solidity.html#binary-packages). Solcjs is not supported.
 
+## Running tests
```

Comment:
I think running most  regular users won't be interested in running tests. Can we put these instructions into a separate file? E.g. README_Developers.md or something like that?

Reply:
Yes, that's a good idea, I can do it now

Reply:
Done and committed